### PR TITLE
add evacuation centre to buildings and places classification

### PR DESCRIPTION
**Pull request by Gustry**
### What does it fix?
* Ticket: #4370 
* Funded by: DFAT
* Description: add evacuation centre to buildings and places classification

### Checklist:
- [ ] Commit messages are descriptive and explain the rationale for changes
- [ ] Add to the changelog in metadata.txt if it's a new feature
- [ ] Unit test for new code added
- [ ] Request someone to review or test your PR

### CODE REVIEW

**Comment by Charlotte-Morgan** [comment]
Hi @Gustry - this looks fine to me. Maybe @lrpower should check this with some of the Pacific program data?
I've made some comments in the files

File: safe/definitions/exposure_classifications.py [hunk 1]
```diff
@@ -274,6 +274,28 @@
...
+            'key': 'evacuation centre',
+            'name': tr('Evacuation centre'),
+            'description': tr(
+                'Evacuation centres provide a place of temporary refuge for '
+                'people evacuated from homes in the vicinity of a disaster '
+                'and for people who may be travelling through the affected '
+                'area but are unable to continue.'),
+            'string_defaults': [
+                'evacuation centre',
+                'refugee',
```
Comment (Charlotte-Morgan) [inline safe/definitions/exposure_classifications.py]: I would not include refugee (person) in this list.  Maybe you meant refuge?

File: safe/definitions/exposure_classifications.py [hunk 2]
```diff
@@ -531,6 +553,28 @@
...
+                'idp camp'
+            ],
+            'osm_downloader': [],
+            'citations': [
+                {
```
Comment (Charlotte-Morgan) [inline safe/definitions/exposure_classifications.py]: If this definition is from a specific source, please cite the source

File: safe/definitions/exposure_classifications.py [hunk 3]
```diff
@@ -274,6 +274,28 @@
                 }
             ]
         },
+        {
+            # IF YOU UPDATE THIS ONE, PLEASE UPDATE THE OTHER ONE IN PLACES TOO
+            'key': 'evacuation centre',
+            'name': tr('Evacuation centre'),
```
Comment (Charlotte-Morgan) [inline safe/definitions/exposure_classifications.py]: To be consistent with other names  this should be Evacuation Centre

**Comment by Charlotte-Morgan** [comment]
Hi @Gustry - this looks fine to me. Maybe @lrpower should check this with some of the Pacific program data?
I've made some comments in the files
BTW - this work is for the Pacific Program - PacSAFE project not DMI Indonesia program

File: safe/definitions/exposure_classifications.py (hunk 2 above)
Comment (Gustry) [reply safe/definitions/exposure_classifications.py]: I know it was weird, but I found this definition on the google page. Then when I clicked the link, the definition wasn't existing anymore on the website. I tried this morning, my history navigation, still the same, I can't find the website again.

**Comment by Gustry** [comment]
I'm will merge this one. I will keep the ticket open for further discussion

**Pull request merged**